streaming/backoff.go

```diff
@@ -12,7 +13,12 @@ type backoffParams struct {
 }
 
 func defaultBackoffHandler(attempts int) time.Duration {
-	jitter := time.Duration(rand.Intn(200)-100) * time.Millisecond                       // ±100ms
+	randInt, err := rand.Int(rand.Reader, big.NewInt(200))
```

Comment:
gosec flag:
```shell
[/Users/ed/dev/luno-go/streaming/backoff.go:15] - G404 (CWE-338): Use of weak random number generator (math/rand or math/rand/v2 instead of crypto/rand) (Confidence: MEDIUM, Severity: HIGH)
    14: func defaultBackoffHandler(attempts int) time.Duration {
  > 15:         jitter := time.Duration(rand.Intn(200)-100) * time.Millisecond                       // ±100ms
    16:         backoff := time.Duration(math.Min(math.Pow(2, float64(attempts)), 60)) * time.Second // Exponential backoff up to 60s
```